Formula/p11-kit.rb

```diff
@@ -0,0 +1,27 @@
+class P11Kit < Formula
+  desc "Library to load and enumerate PKCS#11 modules"
+  homepage "https://p11-glue.freedesktop.org"
+  url "https://p11-glue.freedesktop.org/releases/p11-kit-0.23.2.tar.gz"
+  sha256 "ba726ea8303c97467a33fca50ee79b7b35212964be808ecf9b145e9042fdfaf0"
+
+  depends_on "pkg-config" => :build
+  depends_on "libffi"
```

Comment:
I'd actually go from the documentation at https://p11-glue.freedesktop.org/doc/p11-kit/devel-building.html:

> **libffi** for sharing of PKCS#11 modules between multiple callers in the same process. It is highly recommended that this dependency be treated as a required dependency.

I'd just set this as a dependency - don't make it optional.  It kind of defeats the purpose.

> **libtasn1** is required to build the trust module and code that interacts with certificates.

Since you are setting `--disable-trust-module`, you probably don't need to list `libtasn1` as a dependency...but you might need to also specify `--without-libtasn1` (but I'm not positive).

Alternatively, you could build `--disable-trust-module` when `libtasn1` is not included - and actually build the trust module when `libtasn1` is set as a dependency.  However, you may also need to specify `--with-trust-paths` because there really isn't a good "standard" trust path that I'm aware of on OS X.